iotdb-core/node-commons/src/main/java/org/apache/iotdb/commons/pipe/config/constant/PipeExtractorConstant.java

```diff
@@ -24,6 +24,12 @@ public class PipeExtractorConstant {
   public static final String EXTRACTOR_KEY = "extractor";
   public static final String SOURCE_KEY = "source";
 
+  public static final String EXTRACTOR_DIALECT_KEY = "extractor.dialect";
```

Comment:
Seemingly this is not decided by the model used in creation and is left for the users to fill in?